patterns/aws-vpc-cni-network-policy/versions.tf

```diff
@@ -14,5 +14,9 @@ terraform {
       source  = "hashicorp/kubernetes"
       version = ">= 2.20"
     }
+    null = {
+      source  = "hashicorp/null"
+      version = ">= 3.2"
+    }
```

Comment:
Do we need this, since you replaced the null_resources?

Reply:
This was a typo in a wrong file! I've removed it accordingly!